aws/resource_aws_serverlessapplicationrepository_cloudformation_stack_test.go

```diff
@@ -492,12 +494,14 @@ locals {
   application_account = local.security_manager_accounts[data.aws_partition.current.partition]
 
   security_manager_regions = {
-    "aws"        = "us-east-1",
-    "aws-us-gov" = "us-gov-west-1",
+    %[1]q = %[3]q
+    %[2]q = %[4]q
   }
+
   security_manager_accounts = {
-    "aws"        = "297356227824",
-    "aws-us-gov" = "023102451235",
+    %[1]q = "297356227824"
+    %[2]q = "023102451235"
   }
 }
-`
+`, endpoints.AwsPartitionID, endpoints.AwsUsGovPartitionID, endpoints.UsEast1RegionID, endpoints.UsGovWest1RegionID)
```

Comment:
It's unusual to use these Go constants in config but don't see a good way to use a current data source since neither the partitions nor regions are the current.

Reply:
It might be easiest if this local value generation was moved outside the Terraform configuration into the Go code instead for readability, then the generated ARN was just passed into the configuration. `%[1]q = %[3]q` and `%[1]q = "297356227824"` could be confusing for someone trying to read the configuration for the first (or a much later) time.

Reply:
I agree. Much cleaner this way.